core_contracts/governance/governance.py

```diff
@@ -28,30 +28,127 @@ class Governance(IconScoreBase):
     def __init__(self, db: IconScoreDatabase) -> None:
         super().__init__(db)
         self.addresses = Addresses(db, self)
+        self.vote_execute = VoteActions(db, self)
         self._launch_day = VarDB('launch_day', db, int)
         self._launch_time = VarDB('launch_time', db, int)
         self._launched = VarDB('launched', db, bool)
+        self._votes = DictDB('votes', db, int, depth=3)
+        self._total_voted = DictDB('total_voted', db, int, depth=2)
+        self._minimum_vote_duration = VarDB('min_duration', db, int)
+        self._vote_actions = DictDB('vote_actions', db, str)
+        self._vote_count = VarDB('vote_count', db, int)
 
     def on_install(self) -> None:
         super().on_install()
         self._launched.set(False)
 
     def on_update(self) -> None:
         super().on_update()
+        self._minimum_vote_duration.set(42300)
```

Comment:
I think the number is trying to resemble a day which is actually 43200.

Reply:
Changed to be in day units (1).